docs/current/guides/235290-troubleshooting.md

```diff
@@ -0,0 +1,42 @@
+---
+slug: /235290/troubleshooting
+displayed_sidebar: 'current'
+category: "guides"
+tags: []
+authors: ["Vikram Vaswani"]
+date: "2023-04-27"
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+# Troubleshooting Dagger
+
+This page describes problems you may encounter when using Dagger, and their solutions.
+
+## Dagger pipeline is unresponsive with a BuildKit error
+
+A Dagger pipeline may hang or become unresponsive, eventually generating a BuildKit error such as `buildkit failed to respond` or `container state improper`.
+
+To resolve this error, you must stop and remove the Dagger Engine container and clear unused volumes and the cache.
+
+1. Stop and remove the Dagger Engine container:
+
+  ```shell
+  DAGGER_CONTAINER_NETWORK_NAME=`docker ps --filter "name=^dagger-engine-*" --format '{{.Names}}'`
+  docker stop $DAGGER_CONTAINER_NETWORK_NAME
+  docker rm $DAGGER_CONTAINER_NETWORK_NAME
```

Comment:
Technically, this is the Docker Engine container provisioned via Docker. I would rename this variable to `DAGGER_ENGINE_DOCKER_CONTAINER`. 

I would also use `"$()"` instead of `` for the command. Rationale: https://github.com/koalaman/shellcheck/wiki/SC2006#rationale

Instead of using `docker` subcommand aliases, I would use the namespace equivalent, e.g. `docker container ...`

All put together:

```console
DAGGER_ENGINE_DOCKER_CONTAINER="$(docker container list --all --filter 'name=^dagger-engine-*' --format '{{.Names}}')"
docker container stop "$DAGGER_ENGINE_DOCKER_CONTAINER"
docker container rm "$DAGGER_ENGINE_DOCKER_CONTAINER"
```

Reply:
Added